src/main/java/org/codehaus/mojo/gwt/shell/JavaCommand.java

```diff
@@ -335,8 +335,7 @@ public void execute()
         {
             if ( timeOut > 0 )
```

Comment:
uhm, really not sure why the error is logged and not considered an error if the timeout is enabled, BUT, if the timeout is not enabled the CommandLineTimeOutException cannot be thrown, so looks like current implementation can never throw an JavaCommandException.

You are changing this, so now a timeout will make the test fail, isn't it? At this point, the 'if (timeout>0)' is useless, and actually the log.error is useless too because now the JavaCommendException contains the error message that it handled later. So... I think is better to remove the whole 'if' block